wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/MicroServiceApplication.java

```diff
@@ -32,11 +27,23 @@
                 "de.muenchen.oss.wahllokalsystem.wahlvorstandservice"
         }
 )
-@EnableAutoConfiguration
+@SpringBootApplication(
+        scanBasePackages = {
+                "org.springframework.data.jpa.convert.threeten",
+                "de.muenchen.oss.wahllokalsystem.wahlvorstandservice",
+                "de.muenchen.oss.wahllokalsystem.wls.common.exception",
+                "de.muenchen.oss.wahllokalsystem.wls.common.security"
+        }
+)
 public class MicroServiceApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(MicroServiceApplication.class, args);
     }
 
+    @Bean
+    @ConfigurationProperties
```

Comment:
Nachdem Code habe ich gesucht weil ich wissen wollte warum die Konfiguration geladen wird. Kannst du das Bitte an die Klasse verlagern, ist aus meiner Sicht logischer.

Reply:
siehe commit [4e08898](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/4e08898f8d13750c68d13408257617215c461a1c)